### ISSUE TITLE
calcDeltaG crashes due to units on equilibrator data

### ISSUE THREAD

**Issue by martamatos** [opencobra/MASS-Toolbox#56]
In the toolbox versions after 1.0.6,  importing <a href="http://tinyurl.com/owvg3vl">this</a> file with equilibrator data takes longer than with v 1.0.6, and then when using it to calculate deltaG, e.g.:

rxn = str2mass["ENO: 2pg_c <=> h2o_c + pep_c"]
calcDeltaG[{rxn}, bigg2equilibrator, is -> 0.25 Mole Liter^-1,  pH -> 7]

there are a number of errors related to unknown units.

**Comment by avsastry** [comment]
The v1.1.0 of the toolbox involved shifting the units package from AutomaticUnits (an external package that is no longer actively supported) to the Mathematica internal units package. When I did this, I did not realize that certain calculations take a significantly longer time to compute using the Mathematica units.

The unknown units are not importing because I'm guessing the old file has only units in the AutomaticUnits format, and not in the new format. Unfortunately for now, the only way to get it to work is to move back to an earlier version. I'll look into it further soon.
